packages/insomnia-url/src/querystring.ts

```diff
@@ -64,32 +66,34 @@ module.exports.buildQueryParameter = function(param, strict) {
 
   // Cast number values to strings
   if (typeof param.value === 'number') {
-    param.value += '';
+    param.value = String(param.value);
```

Comment:
Nice catch!

Reply:
![Screenshot_20210324_091947](https://user-images.githubusercontent.com/15232461/112317066-14f26880-8c82-11eb-8d27-f5f102770bde.png)

oh, don't thank me.  TypeScript demands this.  both here with the operation, but also in the `flexibleEncodeComponent` function that follows that only accepts `string | undefined` (and, not number).